src/components/Board/SymbolSearch/SymbolSearch.component.js

```diff
@@ -429,6 +450,9 @@ export class SymbolSearch extends PureComponent {
             options={this.state.symbolSets}
             onChange={this.handleChangeOption}
           />
+          {!this.state.isFetching &&
```

Comment:
It's probably good to use an object for the isFetching state to avoid some flash appearance of the symbolNotFound when there aren't suggestions for some of the requested symbols banks and there are suggestions for other selected ones. like `isFetching{ arasaac: true, globalSymbol: true }` You could use a spread operator when you want to change one of them and preserve the other like `{isFetching{ ...this.state.isFetching, globalSymbol: true }`

Reply:
Makes sense, I made the requested  changes, but instead of a object I used different state vars to avoid the usage of the spread operator